src/winerror.cr

```diff
@@ -1817,7 +1817,7 @@ class WinError < Errno
   ERROR_SXS_XML_E_EXPECTINGCLOSEQUOTE                           = 14045_u32
   ERROR_SXS_XML_E_MULTIPLE_COLONS                               = 14046_u32
   ERROR_SXS_XML_E_INVALID_DECIMAL                               = 14047_u32
-  ERROR_SXS_XML_E_INVALID_HEXIDECIMAL                           = 14048_u32
```

Comment:
This change is invalid.
https://docs.microsoft.com/en-us/windows/desktop/debug/system-error-codes--12000-15999-#error_sxs_xml_e_invalid_hexidecimal

Reply:
![image](https://user-images.githubusercontent.com/988/52168698-67ccd700-272d-11e9-9c46-c112f1e981f4.png)

Reply:
@oprypin so the Microsoft docs have a spelling error? Can't find anything explaining what hexidecimal could be (and if it's really a real word)

Reply:
I think it's part of Windows API to have a constant with this misspelled name.